man/oci-image-tool-validate.1.md

```diff
@@ -22,7 +22,7 @@ oci-image-tool validate \- Validate one or more image files
   Only applicable if type is image or imageLayout.
 
 **--type**=""
-  Type of the file to validate. If unset, oci-image-tool will try to auto-detect the type. One of "imageLayout,image,imageZip,manifest,imageIndex,config"
+  Type of the file to validate. One of "imageLayout,image,manifest,imageIndex,config"
```

Comment:
Can we set a default type if `--type` is unset? It seems odd if it doesn't work without any options.

Reply:
I don't think it's a good idea. Because `--type` is a little special, it's a must be required option but there is not a value could be used for all files, that means it's almost impossible to find a suitable default value.
So, I think let it to be unset will be better.

Reply:
>   there is not a value could be used for all files, that means it's almost impossible to find a suitable default value.

Default is not a value for all files, it's value are most common.  From that I know, it's not a good design from user perspective if a cli requires a `option` .  


And I tested the patch
```
lei@ubuntu:~/images$ oci-image-tool validate busybox/
--type must be set
lei@ubuntu:~/images$ oci-image-tool validate --type imageLayout busybox/
1 errors detected:
busybox/: type "imageLayout" unimplemented
lei@ubuntu:~/images$ oci-image-tool validate --type image busybox/
autodetected image file type is: imageLayout
oci-image-tool: reference "latest": OK
busybox/: OK
Validation succeeded

```
Is the output `autodetected image file type is: imageLayout` necessary and why  use ` --type imageLayout` doesn't work but the autodetected image type is `imageLayout`?`

Reply:
Oh, sorry....  that's my fault. I should remove imageLayout from help and manpage.
In my design, users can only specify `image,manifest,imageIndex,config` to `--type`. 
If we get that users want to validate an image, then we try to detect its type, may be a layout, tar file or zip file.
As you mentioned at https://github.com/opencontainers/image-tools/pull/141#issuecomment-326907714, I try to use imageType to handle all image format.

Reply:
Updated, removed imageLayout. PTAL